plugins/talk-plugin-featured-comments/client/components/ModTag.js

```diff
@@ -51,10 +42,19 @@ export default class ModTag extends React.Component {
       </span>
     ) : (
       <span className={cn(styles.tag, {[styles.featured]: alreadyTagged})}
-        onClick={this.postTag} >
+        onClick={() => this.props.openFeaturedDialog(comment, asset)} >
```

Comment:
Use a bound method like `this.openFeaturedDialog `

Reply:
Done. What's the reason behind it?

Reply:
It's better for performance to use a single instance of this function instead of creating a new one on each render.